### PR TITLE
MOSIP:17379 - Fix the test case on vidcontroller

### PR DESCRIPTION
**Pull request by abuabdul**

### CODE REVIEW

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilities.java [hunk 1]
```diff
@@ -128,7 +128,7 @@
 
     private String mappingJsonString = null;
 
-    private static String regProcessorIdentityJson = StringUtils.EMPTY;
+    private static String regProcessorIdentityJson = "";
```
Comment (vishwa-vyom) [inline resident/resident-service/src/main/java/io/mosip/resident/util/Utilities.java:131]: If the import is changed, why is this line also changed ?

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java [hunk 2]
```diff
@@ -1,198 +1,202 @@
...
+
+    @Autowired
+    private RestTemplate residentRestTemplate;
+
+    @Autowired
+    private Environment env;
+
+    private static final String IDENTITY = "identity";
+    private static final String VALUE = "value";
+    private static String regProcessorIdentityJson = "";
```
Comment (vishwa-vyom) [inline resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java:70]: If util is changed, why is this changed ?

File: resident/resident-service/src/test/resources/bootstrap.properties [hunk 3]
```diff
@@ -27,7 +27,7 @@ mosip.notification.language-type=both
 mosip.notificationtype=SMS|EMAIL
 resident.notification.emails=girish.yarru@mindtree.com
 resident.notification.message=Notification has been sent to the provided contact detail(s)
-config.server.file.storage.uri=http://localhost:8081
+config.server.file.storage.uri=https://dev2.mosip.net/config/resident/mz/develop
```
Comment (vishwa-vyom) [inline resident/resident-service/src/test/resources/bootstrap.properties]: why is this change required for test bootstrap, we cannot assure that this api is always available

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/test/resources/bootstrap.properties (hunk 3 above)
Comment (abuabdul) [reply resident/resident-service/src/test/resources/bootstrap.properties]: The entire tests are like this. It expects such url to connect. Either we need to skip these tests and make it work for now.

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java (hunk 2 above)
Comment (abuabdul) [reply resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java:70]: It was changed due to formatting. The only line change is     private static String regProcessorIdentityJson = "";

**Review by vishwa-vyom — APPROVED** [review]
